### PR TITLE
Separate the documentation pages!

### PR DESCRIPTION
**Pull request by klecki**
## Description:
Make every operator appear as separate function
in the documentation.

Few hacks needed:
1. ~Turns out that it's really hard to get the canonical
    name of the op and schema from the POV of fn API.
    So I added a `_schema_name` member, maybe there
    is better way to be suggested?~

Now available as #3798

2. All the pages are generated into one `operations` directory,
    mainly to not pollute the docs/ during build, but it has
    a side effect of appearing in the url. Do we want to inlcude
    more of it (module name) as a url hierarchy or are we fine as
    it is now? 

3. ~Redirect didn't work, I need a JS/HTML magician.~
This is fine now.

4. The operations table works on the old API access
    and I have no intention of refactoring it.

### Tests
- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [ ] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by klecki** [comment]
!build

File: dali/python/nvidia/dali/fn.py [hunk 1]
```diff
@@ -89,6 +89,7 @@ def fn_wrapper(*inputs, **kwargs):
     fn_wrapper.__name__ = wrapper_name
     fn_wrapper.__qualname__ = wrapper_name
     fn_wrapper.__doc__ = wrapper_doc
+    fn_wrapper._schema_name = op_class.schema_name
```
Comment (mzient) [inline dali/python/nvidia/dali/fn.py:92]: I'd suggest passing the op_class, not just the name - that way we'd have access to whatever properties of the underlying op class we want.
```suggestion
    fn_wrapper._op_class = op_class
```
Also, please add it also to op_wrapper, too. I think this code needs some cleanup after the introduction of debug mode... :\

Comment (klecki) [reply dali/python/nvidia/dali/fn.py:92]: Probably this change or anything related should be made it separate PR not during docs rework.

I am a bit worried about tying the whole class to the fn functions, but those are private members, and who checks the members of the functions?

File: docs/_static/redirect.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// TODO(klecki): doesn't work, how to include it?
```
Comment (jantonguirao) [inline docs/_static/redirect.js:1]: Is this meant to stay?

**Review by jantonguirao — APPROVED** [review]

**Review by prak-nv — APPROVED** [review]

**Comment by klecki** [comment]
!build

**Review by prak-nv — APPROVED** [review]

**Pull request merged**